Libraries/xcsdk/CMakeLists.txt

```diff
@@ -23,3 +23,4 @@ target_include_directories(xcsdk PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/Headers")
 add_executable(xcrun Tools/xcrun.cpp)
 target_link_libraries(xcrun xcsdk util)
 
+install(TARGETS xcrun RUNTIME DESTINATION bin)
```

Comment:
Sure.  I don't know which targets need install(), so your advice there would be appreciated.